CHANGELOG.md

```diff
@@ -28,6 +28,55 @@
   reserved word in Erlang/OTP 27.
   ([Jake Barszcz](https://github.com/barszcz))
 
+- Fault tolerance is now at a statement level instead of a function level in most cases. This means that even if there are errors in a single statement, the compiler will attempt to infer the rest of the function. The exceptions to this are if the right hand side of a use is an invalid function or if the left hand side of an assignment is an invalid pattern.
```

Comment:
What does the last sentence here mean?

Reply:
Originally I didn't know how to handle those 2 cases and left them as intolerant. I think I worked out a better solution so this should no longer be valid. in the case of an invalid pattern we will report and register nothing. in the case of an invalid right hand side of use we'll just mark the entire use as an invalid expression